### PR TITLE
add functionality for claiming Vape from EmissionsManager

### PR DESCRIPTION
**Pull request by parth-15**

### CODE REVIEW

File: src/facets/DiamondManagerFacet.sol [hunk 1]
```diff
@@ -156,6 +159,19 @@ contract DiamondManagerFacet {
         season.rewardTokenBalance = _rewardTokenToDistribute;
     }
 
+    function claimTokensForSeason() external onlyOwner {
+        IEmissionsManager(s.emissionsManager).mintLiquidStaking();
```
Comment (mejiasd3v) [inline src/facets/DiamondManagerFacet.sol:163]:
```suggestion
        IEmissionsManager(s.emissionsManager).mintLiquidMining();
```

File: src/libraries/AppStorage.sol [hunk 2]
```diff
@@ -94,4 +94,6 @@ struct AppStorage {
     address feeToken;
     address stratosphereAddress;
     uint256 reentrancyGuardStatus;
+
+    address emissionsManager;
```
Comment (mejiasd3v) [inline src/libraries/AppStorage.sol:98]: we will deploy a new diamond, can you move this into the "general" section

Comment (parth-15) [reply src/libraries/AppStorage.sol:97]: it's already in general. I have removed the blank line in betwee.

File: src/facets/DiamondManagerFacet.sol (hunk 1 above)
Comment (parth-15) [reply src/facets/DiamondManagerFacet.sol]: I have made this change. I will create separate PR to change function name in EmissionsManager contract.

**Review by mejiasd3v — APPROVED** [review]

**Pull request merged**